docs/content/preview/releases/release-notes/v2.20.md

```diff
@@ -65,6 +65,7 @@ docker pull yugabytedb/yugabyte:2.20.0.0-b76
 * Support for Transactional CDC.
 * Wait-on conflict concurrency control with pg_locks. This feature is especially important to users that have workloads with many multi-tablet statements, which may have multiple sessions contending for the same locks concurrently. Wait-on conflict concurrency control helps avoid a deadlock scenario as well as ensure fairness so a session is not starved by new sessions. [Early Access]
 * Catalog metadata caching to improve YSQL connection scalability and performance with faster warm up times. [Early access]
+* Expand `ALTER TABLE` capabilities by introducing support for `ALTER COLUMN TYPE` operations that involve on-disk changes. This allows a more extensive range of data type modifications, accommodating scenarios such as changing the type itself and not just increasing size. [GA]
```

Comment:
Support for ALTER COLUMN TYPE  with on-disk changes. Expand ALTER TABLE capabilities with a broader range of data type modifications, including changes to the type itself, not just size increments. [GA]